src/components.test.js

```diff
@@ -0,0 +1,35 @@
+import { render } from '@testing-library/react';
```

Comment:
You did a great job implementing the changes given by the previous reviewer. However, you have some missing parts and some errors that you need to handle before approving this project. So, let's get started :)

- [ ] 1. When I run `npm test` I get errors after a few seconds: 
 <img width="764" alt="Screenshot 2022-11-20 at 00 16 32" src="https://user-images.githubusercontent.com/69011963/202867816-53456602-2921-477f-aab1-7662567738b7.png">
 Kindly fix them :)

Reply:
The tests are passing on my end. See screenshot below

![image](https://user-images.githubusercontent.com/94530406/202868740-de5c5d39-0fe8-4b6a-87ba-8f7c6d263601.png)

Reply:
Yes, but wait for it. It's showing up after the test complete.